ddtrace/span.py

```diff
@@ -528,7 +543,7 @@ def _pprint(self):
             ("duration", self.duration),
             ("error", self.error),
             ("tags", dict(sorted(self._meta.items()))),
-            ("metrics", dict(sorted(self.metrics.items()))),
+            ("_metrics", dict(sorted(self._metrics.items()))),
```

Comment:
The key for this item should remain as is for the sake of `pprint`:

```suggestion
            ("metrics", dict(sorted(self._metrics.items()))),
```